src/tests/VoxelGridTest.cpp

```diff
@@ -105,6 +108,41 @@ void VoxelGrid::testVoxelGridWithVHACD() {
   // Turn off visualization
   simulator_->setStageVoxelizationDraw(false, "Boundary");
 }
+
+void VoxelGrid::testVoxelUtilityFunctions() {
+  // configure and intialize Simulator
+  auto simConfig = esp::sim::SimulatorConfiguration();
+  simConfig.activeSceneName = Cr::Utility::Directory::join(
+      SCENE_DATASETS, "habitat-test-scenes/skokloster-castle.glb");
+  simConfig.enablePhysics = true;
+  simConfig.frustumCulling = true;
+  simConfig.requiresTextures = true;
+
+  auto simulator_ = esp::sim::Simulator::create_unique(simConfig);
+
+  // Voxelize the scene with resolution = 1,000,000 and make asserts
+  const int resolution = 1000000;
+  simulator_->createStageVoxelization(resolution);
+  auto voxelization = simulator_->getStageVoxelization();
+
+  // Generate Euclidean SDF for the voxelization
+  esp::geo::generateEuclideanDistanceSDF(voxelization, "ESDF");
```

Comment:
I'd like to see some verification of the results here in case the logic changes. Ideally an import/export would be used, but a few golden values would be good enough for now.